README.md

```diff
@@ -77,7 +78,7 @@ Fork and send a Pull Request. Thanks!
 
 ## Copyright
 
-Copyright (c) *2022* *The Foreman developers*
+Copyright (c) *2023* *The Foreman developers*
```

Comment:
It's more common to make it 2022-2023.

Reply:
I changed it, I think that makes sense. I found it not being very consistent in the Foreman world, though.